mkosi

```diff
@@ -22,6 +22,9 @@ from enum import Enum
 
 __version__ = '1'
 
+if sys.version_info[1] < 5:
```

Comment:
That's unpythonic, e.g. python 2.7 passes that test. (And a hypothetical python 4.0 would also fail the test...). I'd prefer a check like:
```python
if sys.version_info < (3, 5):
```